branch_test/oracle.py

```diff
@@ -0,0 +1,87 @@
+import numpy as np
+from scipy.stats import norm, iqr
+from sklearn.neighbors.kde import KernelDensity
+from sklearn.metrics import auc, roc_curve
+import string
+
+
+class BinaryRSVPOracle(object):
```

Comment:
I've seen you implement this a few times. It could be moved somewhere.

Reply:
I implemented a more general simulator using RSVP data for the debugging during August. I totally agree need a place for this implementation, which can serve as a broad demo of the system.